Multiple await keywords on a single line (#27019)
dotnet/docs: docs/standard/garbage-collection/implementing-disposeasync.md

:::code language="csharp" source="snippets/dispose-async/ExampleUsingDeclarationProgram.cs":::

## Multiple await keywords in a single line

Sometimes the `await` keyword may appear multiple times within a single line. For example, consider the following code:

```csharp
await using var transaction = await context.Database.BeginTransactionAsync(token);
```

In the preceding example:

- The <xref:System.Data.Common.DbConnection.BeginTransactionAsync%2A> method is awaited.

> - The return type is <xref:System.Data.Common.DbTransaction>, which is an implementation of `IAsyncDisposable`.

Reviewer: ```suggestion
- The return type is <xref:System.Data.Common.DbTransaction>, which implements `IAsyncDisposable`.
```
